deepfence_frontend/apps/dashboard/src/features/common/data-component/searchClustersApiLoader.tsx

```diff
@@ -35,7 +35,10 @@ export const searchClustersApiLoader = async ({
         filters: {
           compare_filter: null,
           contains_filter: {
-            filter_in: null,
+            filter_in: {
+              pseudo: [false],
+              active: [true],
```

Comment:
It would make sense to show inactive nodes in the filters for example in the scan history page, we should make this configurable. Also if we are adding this here, we also need to add it for container and container images filter dropdowns.

Reply:
Can you please check with backend team and make sure?

Reply:
@manV @harishshg `active: [true]` is required in diagnosis page.
There are probably other cases where we need it.
There are also case where we don't need it.
So it has to be configurable.

Reply:
`pseudo: [false]` is required in all cases.

Reply:
additional filters for all searches can be made as configurable

Reply:
added active filter flag at searchList components level for hosts, clustors, images and imageContainers.